ci-operator/config/jboss-fuse/fuse-xpaas-qe-container/jboss-fuse-fuse-xpaas-qe-container-main__jboss-fuse-ocp4.14-lp-interop.yaml

```diff
@@ -0,0 +1,43 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: jboss-fuse-interop-aws
+  cron: 0 6 * * 1
```

Comment:
Should cron not be set to 0 6 25 12 * now.

Reply:
Hi @smatula can you be more clear? Please feel free to suggest the correction. "25" is it a valid value for cron expression?

Reply:
Hi @smongiar We have [another job](https://github.com/openshift/release/blob/master/ci-operator/config/rhpit/interop-tests/rhpit-interop-tests-main__weekly_trigger.yaml) in place now to trigger these scenario every week, so please set it `0 6 25 12 *` so that it can only trigger once a year from this cron value.